src/Xamarin.Android.Tools.Bytecode/Kotlin/KotlinFixups.cs

```diff
@@ -286,6 +341,8 @@ static void FixupField (FieldInfo? field, KotlinProperty metadata)
 				return method;
 			}
 
+			// Theoretically this should never be hit, but who knows. At worst, it just means
```

Comment:
Should we emit a debug message or warning here?  Pity `Action<TraceLevel, string>` isn't already used here.

Reply:
Added a debug message to match the rest of Kotlin debugging messages. A warning probably wouldn't be appropriate because it's not actionable.  There's nothing a user can do to fix it.